setup.py

```diff
@@ -34,7 +34,8 @@ def get_version(rel_path):
     "sceptre-cmd-resolver>=1.1.3,<2",
     "sceptre-file-resolver>=1.0.4,<2",
     "six>=1.11.0,<2.0.0",
-    "networkx>=2.6,<2.7"
+    "networkx>=2.6,<2.7",
+    "urllib3==1.25.11"
```

Comment:
this could cause no end of dependency hell for our users. We shouldn't ever pin a specific version like this. Why are you making this change?

Reply:
desperate times calls for desperate measures, https://github.com/Yelp/elastalert/issues/3037#issuecomment-726025270

Reply:
Yeah, but this is going to conflict with so many other peoples' environments. I don't understand how that issue is related to our own environments. This seems like it could be a major headache.

Reply:
i thought everybody runs in a virtualenv? 😄   Do you have a suggestion @jfalkenstein ?

Reply:
Especially since this is really a hotfix for an old version, I'd be satified with `urllib3<2.0`

Reply:
done